qa/suites/deepsea/build-validation/ses_rgw_zones/cluster/roles.yaml

```diff
@@ -1,4 +1,4 @@
 roles:
   - [client.salt_master, mon.a, mgr.x]
-  - [osd.0]
-  - [osd.1]
+  - [osd.0, osd.1]
+  - [osd.2, osd.3]
```

Comment:
This change doesn't have any effect, does it? In the deepsea suite, "osd" role means: "create OSDs on all available disks". So there is no point in having multiple `osd` roles per node.